File: hdcd_source_spa_react_ts_springboot_uuppsd (2)/hdcd_source_spa_react_ts_springboot/ch1305/src/main/java/org/hdcd/domain/result/Board.java
package org.hdcd.domain.result;

import java.time.LocalDateTime;

public class Board {

	private int boardNo;
	private String title;
	private String content;
	private String writer;
	private LocalDateTime regDate;
	
	@Override
	public String toString() {
		return "Board(boardNo=" + boardNo + ", title=" + title + ", content=" + content + ", writer=" + writer + ", regDate=" + regDate + ")";
	}
	
}

File: hdcd_source_spa_react_ts_springboot_uuppsd (2)/hdcd_source_spa_react_ts_springboot/ch0706/src/modules/board.ts
import { createReducer } from "typesafe-actions";
import { Board } from "../App"

export interface BoardState {
  loading: { FETCH: boolean, FETCH_LIST: boolean};
  board: Board;
  boards: Board[];
  error: any;
}

const initialState: BoardState = {
  loading: {
    FETCH: false,
    FETCH_LIST: false,
  },
  board: { boardNo: '', title: '', writer: '', content: '', regDate: '' },
  boards: [],
  error: null,
};

const board = createReducer(
  initialState,
  {},
);

export default board;
